ocean_lib/ocean/test/test_ocean_assets.py

```diff
@@ -39,43 +40,56 @@ def test_register_asset(publisher_ocean_instance, publisher_wallet, consumer_wal
     assert ocn.assets.resolve(invalid_did) is None
 
 
-@pytest.mark.skip(reason="TODO: update function on OceanAssets class")
-def test_update(publisher_ocean_instance, publisher_wallet, consumer_wallet, config):
-    ocn = publisher_ocean_instance
-    block = ocn.web3.eth.block_number
-    alice = publisher_wallet
-    bob = consumer_wallet
+def test_update(publisher_ocean_instance, publisher_wallet, config, metadata=None):
+    """Helper function for asset update based on ddo_sa_sample.json."""
+    ocean = publisher_ocean_instance
+    erc20_data = ErcCreateData(
+        template_index=1,
+        strings=["Datatoken 1", "DT1"],
+        addresses=[
+            publisher_wallet.address,
+            publisher_wallet.address,
+            ZERO_ADDRESS,
+            get_address_of_type(config, "Ocean"),
+        ],
+        uints=[ocean.web3.toWei("0.5", "ether"), 0],
+        bytess=[b""],
+    )
 
-    ddo = create_asset(ocn, alice, config)
+    if not metadata:
+        metadata = {
+            "created": "2020-11-15T12:27:48Z",
+            "updated": "2021-05-17T21:58:02Z",
+            "description": "Sample description",
+            "name": "Sample asset",
+            "type": "dataset",
+            "author": "OPF",
+            "license": "https://market.oceanprotocol.com/terms",
+        }
+    data_provider = DataServiceProvider
+    file1_dict = {"type": "url", "url": "https://url.com/file1.csv", "method": "GET"}
```

Comment:
I think we are overusing this in tests. You can just create a UrlFile with url_file.url = ...
instead of creating this from_dict everytime.

but what if we update the files themselves? Or the service structure? Then it's not so simple. Here we are basically attaching a random file, but what if we want to change the files as part of the update? Or the service structure? I don't think the task is complete just by setting the metadata. Can you address this on Slack?